packs/core-moves/quick-guard.json

```diff
@@ -11,40 +11,40 @@
         "type": "attack",
         "traits": [
           "shield",
-          "exhaust",
-          "interrupt-4",
-          "pp-updated"
+          "pp-updated",
+          "interrupt-7"
         ],
         "range": {
-          "target": "emanation",
+          "target": "creature",
           "distance": 4,
           "unit": "m"
         },
         "cost": {
           "activation": "complex",
-          "powerPoints": 3,
+          "powerPoints": 5,
           "trigger": "<p>The user or an ally in range is targeted by a [Priority X] Action.</p>"
         },
         "category": "status",
-        "power": null,
-        "accuracy": null,
         "types": [
           "fighting"
         ],
-        "description": "<p>Trigger: The user or an ally in range is targeted by a [Priority X] Action.</p><p>Effect: The triggering attack fails to hit, and additional attacks that use Priority that would hit also fail to hit until until the round ends. These attacks that would fail to hit do not activate on-miss effects.</p>",
-        "contestType": "",
-        "contestEffect": "",
-        "free": false,
-        "slot": null
+        "description": "<p>Trigger: The user or an ally in range is targeted by a [Interrupt X] Action.</p><p>Effect: The triggering attack fails to hit the target, and additional [Interrupt X] attacks that would hit the target, the user, or other allies within range, also fail to hit until the end of user's next Activation. These attacks that would fail to hit do not activate on-miss effects.</p><p>On resolution, the user gains @Affliction[disabled] 5 on this Attack.</p>",
```

Comment:
```suggestion
        "description": "<p>Trigger: The user or an ally in range is targeted by a @Trait[interrupt-x] Action.</p><p>Effect: The triggering attack fails to hit the target, and additional @Trait[interrupt-x] attacks that would hit the target, the user, or other allies within range, also fail to hit until the end of user's next Activation. These attacks that would fail to hit do not activate on-miss effects.</p><p>On resolution, the user gains @Affliction[disabled] 5 on this Attack.</p>",
```